### PR TITLE
Display error message on web app

### PR DESCRIPTION
**Pull request by KomalBhadani27**
- Created a separate error screen html to display on iframe if no stamp generated
- Show message on raw code file if error screen loaded in iframe

Image

### CODE REVIEW

File: error.py [hunk 1]
```diff
@@ -35,5 +35,5 @@ def __init__(self, message, failure_source):
 class WebsiteConnectionError(Error):
```
Comment (detel) [inline error.py:35]: Can you change the base error to class StampifierError(Exception):?

File: api/server.py [hunk 2]
```diff
@@ -50,4 +50,5 @@ def stampify_url():
     try:
         return _stampifier.stampify().stamp_html
     except Error as err:
-        return err.message
+        return render_template('error_screen.html',
```
Comment (detel) [inline api/server.py:53]: Please this pylint error
`api/server.py:8:0: E0611: No name 'Error' in module 'error' (no-name-in-module)`

File: data_models/website.py [hunk 3]
```diff
@@ -14,7 +14,7 @@ class Website:
 
     def __init__(self, url):
         self.url = url_utils.valid_url(url)
-        self.is_valid = True if url else False
+        self.is_valid = True if self.url else False
```
Comment (detel) [inline data_models/website.py:17]: data_models/website.py:19:4: C0116: Missing function or method docstring (missing-function-docstring)
data_models/website.py:32:-1: W0105: String statement has no effect (pointless-string-statement)
data_models/website.py:39:4: C0116: Missing function or method docstring (missing-function-docstring)

File: extraction/extractor.py [hunk 4]
```diff
@@ -6,6 +6,7 @@
 
 import requests
 from bs4 import BeautifulSoup, Comment
+from requests.exceptions import InvalidSchema, MissingSchema
```
Comment (detel) [inline extraction/extractor.py:9]: extraction/extractor.py:11:0: E0611: No name 'NoneTypeMarkupError' in module 'error' (no-name-in-module)
extraction/extractor.py:11:0: E0611: No name 'WebsiteConnectionError' in module 'error' (no-name-in-module)
extraction/extractor.py:11:0: E0611: No name 'WebsiteNotStampifiableError' in module 'error' (no-name-in-module)

File: api/server.py (hunk 2 above)
Comment (KomalBhadani27) [reply api/server.py:53]: It worked for me. :/
`$ pylint api/

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
`

File: data_models/website.py (hunk 3 above)
Comment (KomalBhadani27) [reply data_models/website.py:17]: My bad! Forgot to do it for data models.

File: extraction/extractor.py (hunk 4 above)
Comment (KomalBhadani27) [reply extraction/extractor.py:9]: Works fine for me :/
$ pylint extraction/

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

File: extraction/extractor.py [hunk 5]
```diff
@@ -73,8 +74,8 @@ def extract_html(self):
         try:
             # To request Html from URL
             file = REQUEST_SESSION.get(self.url).text
-        except requests.exceptions.ConnectionError:
-            raise WebsiteConnectionError(self.url)
+        except (ConnectionError, InvalidSchema, MissingSchema):
+            raise WebsiteConnectionError()
```
[the remark this answers is not in the document]
Comment (KomalBhadani27) [reply extraction/extractor.py:78]: Updated! But I feel we should not have it as suggested by pylint also that it's a broad-except. Let me know if it's okay to remove that.

File: extraction/extractor.py (hunk 4 above)
Comment (detel) [reply extraction/extractor.py:9]: Did you change your pylintrc file?

File: extraction/extractor.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (detel) [reply extraction/extractor.py:78]: Okay, remove it.

File: extraction/extractor.py (hunk 4 above)
Comment (KomalBhadani27) [reply extraction/extractor.py:9]: No

**Pull request merged**